pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ControllerPeriodicTasksIntegrationTests.java

```diff
@@ -101,6 +107,9 @@ public void setUp() throws Exception {
     controllerConf.setStatusCheckerFrequencyInSeconds(PERIODIC_TASK_FREQ_SECONDS);
     controllerConf.setRealtimeSegmentRelocationInitialDelayInSeconds(PERIODIC_TASK_INITIAL_DELAY_SECONDS);
     controllerConf.setRealtimeSegmentRelocatorFrequency(PERIODIC_TASK_FREQ);
+    controllerConf.setBrokerResourceValidationInitialDelayInSeconds(PERIODIC_TASK_FREQ_SECONDS);
+    controllerConf.setBrokerResourceValidationFrequencyInSeconds(PERIODIC_TASK_FREQ_SECONDS);
+    controllerConf.setControllerPort("9550");
```

Comment:
That's because I got the following exception messages if I use the default one:
```
javax.ws.rs.ProcessingException: Failed to start Grizzly HTTP server: Address already in use

	at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpServerFactory.createHttpServer(GrizzlyHttpServerFactory.java:270)
	at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpServerFactory.createHttpServer(GrizzlyHttpServerFactory.java:93)
	at org.apache.pinot.controller.api.ControllerAdminApiApplication.start(ControllerAdminApiApplication.java:85)
	at org.apache.pinot.controller.ControllerStarter.setUpPinotController(ControllerStarter.java:324)
	at org.apache.pinot.controller.ControllerStarter.start(ControllerStarter.java:201)
	at org.apache.pinot.controller.helix.ControllerTest.startControllerStarter(ControllerTest.java:164)
...
```